osvr/RenderKit/RenderManagerD3D.cpp

```diff
@@ -290,5 +292,120 @@ namespace renderkit {
       return true;
     }
 
+    bool RenderManagerD3D11::GetTimingInfo(size_t whichEye,
```

Comment:
Did this actually change or was this actually added in this pull request? I thought D3D11 backend already had a GetTimingInfo implementation.